packages/paste-design-tokens/formatters/__tests__/gatsby.json.spec.ts

```diff
@@ -1,28 +0,0 @@
-import * as theo from 'theo';
-import {resolve} from 'path';
-import {gatsbyJsonTokenFormat} from '../gatsby.json';
-
```

Comment:
question: why are these formatter tests being deleted?

Reply:
For the additional context: We're moving to use the token formatter, and no longer using the gatsby token format. I had thought I had removed all refs in a prev commit to the new tokens page branch but missed the test. Resolving that here so the CI checks pass.

Reply:
For the additional context: We're moving to use the "generic" token format, and no longer using the gatsby token format. I had thought I had removed all refs in a prev commit to the new tokens page branch but missed the test. Resolving that here so the CI checks pass.